<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GraphApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: rgba(200, 200, 200, 0.95);
    border: 1px solid #e0e0e0;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" rowspan="4">
      <widget class="QGraphicsView" name="graphicsView">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonFunction">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
       </property>
       <property name="text">
        <string>Применить
алгоритм</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonClear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
       </property>
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButtonRules">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
       </property>
       <property name="text">
        <string>Как
пользоваться?</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonExit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton { background: #00b4d8; color: #222222; border: 1px solid #0096c7; border-bottom: 3px solid #0096c7; border-radius: 8px; padding: 10px 20px; font: bold 14px; } QPushButton:hover { background: #48cae4; border-bottom: 3px solid #00a8e8; } QPushButton:pressed { background: #0096c7; border-bottom: 1px solid #0077b6; margin-top: 2px; }</string>
       </property>
       <property name="text">
        <string>Выход</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
